src/ACEtk/Xsdir/test/Xsdir.test.cpp

```diff
@@ -196,6 +221,30 @@ std::string chunk() {
     " 92000.00p   233.000000 file3 0 1 4 3000\n";
 }
 
+std::string chunkWithCommentLines() {
+
+  return
+    "# this is a comment line\n"
+    "datapath=/some/path/to/Data\n"
+    "# this is a comment line\n"
+    "# this is a comment line\n"
+    "atomic weight ratios\n"
+    "       1   1.00000000\n"
+    "    1000   0.99931697    1001   0.99916733    1002   1.99679968    1003   2.99013997\n"
+    "                         1004   3.99320563\n"
+    "# this is a comment line\n"
+    "   92000 235.98412800   92235 233.02478975 1092235 233.02478975\n"
+    "\n"
+    "# this is a comment line\n"
+    "directory\n"
+    "# this is a comment line\n"
+    " 92234.00c   234.000000 file 0 1 5 1000 0 0 2.530000e-08\n"
+    " 92235.00c   235.000000 file2 0 1 3 2000 0 0 2.530000e-08 ptable\n"
```

Comment:
There is no test of a continue line.

Reply:
Good point. I will add one.

Reply:
There actually is one in the XsdirEntry test - but it never hurts to add one to Xsdir too.